server/hotrod/src/main/scala/org/infinispan/server/hotrod/OperationStatus.scala

```diff
@@ -30,6 +30,7 @@ object OperationStatus extends Enumeration {
    val OperationTimedOut = Value(0x86) // todo: test
    val NodeSuspected = Value(0x87)
    val IllegalLifecycleState = Value(0x88)
+   val Interrupted = Value(0x89)
```

Comment:
My initial though was to piggyback on IllegalLifecycleState, but an ```InterruptedException``` could be throw in other situations apart from server going down. For example, a long running job (script?) that gets explicitly cancelled by the user